website/docs/d/private_link_service.html.markdown

```diff
@@ -0,0 +1,133 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_private_link_service"
+sidebar_current: "docs-azurerm-datasource-private-link-service"
+description: |-
+  Gets information about an existing Private Link Service
+---
+
+# Data Source: azurerm_private_link_service
+
+Use this data source to access information about an existing Private Link Service.
+
+
+## Private Link Service Usage
+
+```hcl
+data "azurerm_private_link_service" "example" {
+  resource_group_name = "acctestRG"
+  name                = "acctestpls"
+}
+
+output "private_link_service_id" {
+  value = "${data.azurerm_private_link_service.example.id}"
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the private link service.
+
+* `resource_group_name` - (Required) The name of the resource group.
+
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `location` - Resource location.
+
+* `alias` - The alias of the private link service.
+
+* `auto_approval` - One `auto_approval` block defined below.
+
+* `fqdns` - The list of Fqdn.
```

Comment:
As below, could we expand on this description?

Reply:
Removed...